src/Data/List/Relation/Binary/BagAndSetEquality.agda

```diff
@@ -388,10 +388,10 @@ drop-cons {x = x} {xs} {ys} x∷xs≈x∷ys =
     index-of (to xs≈ys (proj₂
       (from (Fin-length xs) (to (Fin-length xs) (z , p)))))   ≡⟨⟩
 
-    index-of (proj₂ (Prod.map id (to xs≈ys)
+    index-of (proj₂ (Product.map id (to xs≈ys)
```

Comment:
May as well refactor to map_1 while editing this?

Reply:
I don't have this line in my version: wrong branch?

Reply:
Actually, I get typechecking errors if I do that!
Maybe I'm missing something?

Reply:
You're missing that @JacquesCarette totally _meant_ to say map_2, I think

Reply:
Right...